Jenkinsfile

```diff
@@ -35,25 +35,39 @@ pipeline {
                         env.awsTokenId = 'aws-sandbox-token'
                         env.awsProfile = 'uoa-sandbox'
                         env.awsAccountId = '416527880812'
+                        withCredentials([
+                            file(credentialsId: "cer-graphql-credentials-sandbox",variable:"credentialsfile")
+                        ]) {
+                            dir("cer-graphql"){
+                                sh "cp $credentialsfile .env"
+                            }                        
+                        }
                     } else if (BRANCH_NAME == 'nonprod') {
                         echo 'Setting variables for nonprod deployment'
                         env.awsCredentialsId = 'aws-its-nonprod-access'
                         env.awsTokenId = 'aws-its-nonprod-token'
                         env.awsProfile = 'uoa-its-nonprod'
-
+                        env.awsAccountId = 'uoa-nonprod-account-id'
                     } else if (BRANCH_NAME == 'prod') {
                         echo 'Setting variables for prod deployment'
                         env.awsCredentialsId = 'uoa-its-prod-access'
                         env.awsTokenId = 'uoa-its-prod-token'
                         env.awsProfile = 'uoa-its-prod'
-
+                        env.awsAccountId = 'uoa-prod-account-id'
                     } else {
                         echo 'You are not on an environment branch, defaulting to sandbox'
                         BRANCH_NAME = 'sandbox'
                         env.awsAccountId = '416527880812'
                         env.awsCredentialsId = 'aws-sandbox-user'
                         env.awsTokenId = 'aws-sandbox-token'
                         env.awsProfile = 'uoa-sandbox'
+                        withCredentials([
```

Comment:
Also, regardless of where you put this stuff (even if it stays here), might as well put in the blank variables for `nonprod`/`prod` envs too 👍

Reply:
I am using `BRANCH_NAME` in the new step to determine which credentials file to load, which means the nonprod/prod envs would use the correct corresponding credentials file. Those files will need to be added to Jenkins credentials store as we implement those envs.